README.rst

```diff
@@ -802,6 +802,61 @@ Mobile Access Technology in use
 .. figure:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/access-technology.png
   :align: center
 
+Iperf
+~~~~~
+
++--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
@Aryamanz29 don't make these tables too wide. As you can observe here it is difficult to read. Please break the content into multiple lines.